bigframes/operations/aggregations.py

```diff
@@ -146,6 +205,24 @@ def skips_nulls(self):
     def handles_ties(self):
         return True
 
+    def output_type(self, *input_types: dtypes.ExpressionType):
+        if isinstance(self.bins, int) and (self.labels is False):
+            return dtypes.INT_DTYPE
```

Comment:
do you have a chance to test it? Is it `FLOAT_DTYPE`?

Reply:
yes, the test suite automatically validates all these derivations now